po/pt_BR.po

```diff
@@ -1,199 +1,234 @@
-# Brazilian Portuguese translations for Extension Manager.
+# Brazilian Portuguese translation for Extension Manager.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the extension-manager package.
-# Douglas Silva <douglasilvadev@gmail.com>, 2023.
+# Douglas Silva <douglasilvadev@gmail.com>, 2024.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: extension-manager\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-28 14:17+0200\n"
-"PO-Revision-Date: 2023-03-20 16:51-0300\n"
+"POT-Creation-Date: 2023-11-22 00:48+0200\n"
+"PO-Revision-Date: 2024-01-02 15:19-0300\n"
 "Last-Translator: Douglas Silva <douglasilvadev@gmail.com>\n"
 "Language-Team: \n"
 "Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"X-Generator: Poedit 3.1.1\n"
+"X-Generator: Poedit 3.4\n"
 
 #: data/com.mattjakeman.ExtensionManager.desktop.in.in:3
 #: data/com.mattjakeman.ExtensionManager.metainfo.xml.in.in:6
 msgid "@app_title@"
 msgstr "@app_title@"
 
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:11
+msgid "Manage GNOME Shell Extensions"
+msgstr "Gerencie Extensões do GNOME Shell"
+
+#. Translators: Do not translate
+#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
+msgid "extension;manager;"
+msgstr "extensão;gerenciador;"
```

Comment:
This string is used to search for the application by its untranslated name since the name of the desktop entry is always translated.

```suggestion
#. Translators: Do not translate
#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
msgid "extension;manager;"
msgstr ""extension;manager;"
```

or

```suggestion
#. Translators: Do not translate
#: data/com.mattjakeman.ExtensionManager.desktop.in.in:12
msgid "extension;manager;"
msgstr ""extension;manager;extensão;gerenciador;"
```

Reply:
I didn't pay attention to the string comment, my bad, but let's keep the default. Peace out !